core/src/main/resources/error/error-classes.json

```diff
@@ -757,6 +757,16 @@
       "The expression <sqlExpr> cannot be used as a grouping expression because its data type <dataType> is not an orderable data type."
     ]
   },
+  "HLL_INVALID_LOG_K" : {
+    "message" : [
+      "Invalid call to <function>; the log K value must be between <min> and <max>, inclusive: <value>."
+    ]
+  },
+  "HLL_UNION_DIFFERENT_LOG_K" : {
```

Comment:
Again: "LOG" -> "LG" or all in one "LGK".

Reply:
Done.